responsibleai/vision/components/rai_vision_insights/spec.yaml

```diff
@@ -81,7 +81,7 @@ outputs:
   ux_json:
     type: path
 code: ../src
-environment: azureml://registries/azureml/environments/responsibleai-vision-ubuntu20.04-py38-cpu/versions/21
+environment: azureml://registries/azureml/environments/responsibleai-vision-ubuntu20.04-py38-cpu/versions/22
```

Comment:
not sure if this version bump is actually necessary - will check after this is merged